test/parallel_api/ranges/std_ranges_mismatch.pass.cpp

```diff
@@ -15,6 +15,14 @@
 
 #include "std_ranges_test.h"
 
+#if _ENABLE_STD_RANGES_TESTING
+namespace test_std_ranges
+{
+template<>
+constexpr bool supress_dangling_iterators_check<std::remove_cvref_t<decltype(oneapi::dpl::ranges::mismatch)>> = true;
```

Comment:
`mismatch` returns 2 borrowed iterators. Was it necessary to suppress the check?